test/unit/specs/components/dropdown.spec.js

```diff
@@ -23,18 +23,19 @@ describe('Dropdown', () => {
     expect(dropdownWrapper.classes()).not.toContain('visible')
   })
 
-  it('should be visible when the container is focused', async () => {
-    const wrapper = render(Dropdown, options)
-    await wrapper.trigger('focus')
-    const dropdownWrapper = wrapper.get('[role="menu"]')
-    expect(dropdownWrapper.classes()).toContain('visible')
-  })
-
-  it('should be visible when the dropdown items are focused', async () => {
-    const wrapper = render(Dropdown, options)
-    const dropdownItem = wrapper.get('#dropdown-item')
-    await dropdownItem.trigger('focus')
-    const dropdownWrapper = wrapper.get('[role="menu"]')
-    expect(dropdownWrapper.classes()).toContain('visible')
-  })
+  // TODO: fix these focus tests
```

Comment:
Oh no! Did upgrading node break these?

Reply:
I suspect something is happening so that the await is not enough for the change from triggering `focus` to take effect

Reply:
```suggestion
```

Reply:
Well, I reverted the node version to 14, and recreated the package-lock.json, and the error went away. Sorry for this inconvenience, @sarayourfriend